packages/gp2-components/src/molecules/UserMenu.tsx

```diff
@@ -0,0 +1,43 @@
+import { css } from '@emotion/react';
+import { logout } from '@asap-hub/routing';
+
+import { NavigationLink, pixels, logoutIcon } from '@asap-hub/react-components';
+
+const { perRem, vminLinearCalc, mobileScreen, largeDesktopScreen } = pixels;
+
+const containerStyles = css({
+  minWidth: '312px',
```

Comment:
Magic

Reply:
Is there any other way we can define this?